BLSQ/iaso
# IA-3360-IA-3361-IA-3362: Form submissions versions history - UX improvements

Explain what problem this PR is resolving
- Highlight what modifications have been made from a form to another
- Use XLS Labels and in slight grey the technical names 
- Sort questions in the ODK form order when comparing forms
Related JIRA tickets : [IA-3360](https://bluesquare.atlassian.net/browse/IA-3360), [IA-3361](https://bluesquare.atlassian.net/browse/IA-3361), [IA-3362](https://bluesquare.atlassian.net/browse/IA-3362)
## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Create a custom GET method to get submission log
- Add possible_fields in the dictionary return by instance/logs to be able to sort Sort questions in the ODK form
- Highlight a row when there is a difference between logs values
- 
## How to test
- Create a submission using enketo and edit it
- visite this link http://localhost:8081/dashboard/forms/compareInstanceLogs/accountId/{accountId}/instanceIds/{submissionId}
- Check if differences are highlighted and questions are sorted like in odk form
## Print screen / video
[Screencast from 2024-09-12 11-15-24.webm](https://github.com/user-attachments/assets/e2fc58da-2b29-486c-9e6c-29bd041ebcb3)

## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3360]: https://bluesquare.atlassian.net/browse/IA-3360?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IA-3361]: https://bluesquare.atlassian.net/browse/IA-3361?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IA-3362]: https://bluesquare.atlassian.net/browse/IA-3362?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
